test/TestCase.h

```diff
@@ -31,6 +31,43 @@
 namespace solidity::frontend::test
 {
 
+/**
+ * A reader for test case data file, which parses source, settings and (optionally) simple expectations.
+ */
+class TestCaseReader
+{
+public:
+	TestCaseReader() = default;
+	TestCaseReader(std::string const& _filename);
+
+	std::map<std::string, std::string> sources() { return m_sources; }
+	std::string source();
+	std::size_t lineNumbers() { return m_lineNumbers; }
+	std::map<std::string, std::string> settings() { return m_settings; }
+	std::ifstream& streamAtExpectations();
+
+	std::string simpleExpectations();
+
+	bool hasSetting(std::string const& _name) const;
+	bool boolSetting(std::string const& _name, bool _defaultValue);
+	unsigned long ulongSetting(std::string const& _name, unsigned long _defaultValue);
```

Comment:
We usually use explicit bit width for types (unless it is irrelevant).

Reply:
It is `size_t` in this case, see 6c9736b.